### ISSUE TITLE
On first Telescope frecency there is a A pretyped in telescope

### ISSUE THREAD

**Issue by fishman** [nvim-telescope/telescope-frecency.nvim#270]
I don't know how to debug this, but whenever I do :Telescope frecency<CR> on the first vim start it shows an 'A' character prefilled in the Telescope window. Is there any way to see what that could be from?

**Comment by fishman** [comment]
```
      vim.keymap.set('n', '<leader>fr', require('telescope').extensions.frecency.frecency, { desc = '[F]ind Recent Files ("." for repeat)' })
```

After starting it manually with `:Telescope frecency` it asked me to remove 12 entries from the frecency database, and after agreeing to it the issue disappeared.

**Comment by delphinus** [comment]
It might output some escape sequences with `vim.ui.select` dialog and you've seen a part of that: `A`, maybe.

Hmm……, I am thinking to disable this dialog in default for avoiding such annoyance.

**Comment by solson** [comment]
I also started noticing this behavior recently, and starting it manually with `:Telescope frecency` doesn't seem to help - that also just opens it with "A" pretyped (if it's the first run).

**Comment by solson** [comment]
I did some `git bisect`ing and found "[673585ee997b40d2600eb86c3693d552e4f5d79f](https://github.com/nvim-telescope/telescope-frecency.nvim/commit/673585ee997b40d2600eb86c3693d552e4f5d79f) is the first bad commit".

Specifically, if I run `:Telescope frecency` before that commit, I get a popup saying `[Telescope-Frecency] remove 16 entries from database?`, but if I run `:Telescope frecency` any time after that commit, I get no popup and I get the prefilled `A`.

**Comment by solson** [comment]
On a separate note, I never understood what that popup was for, back when it worked. It doesn't specify what's special about the 16 entries it's asking me to delete, or why I would want to do so.

It seems like I will most likely either always say Yes or always say No, so maybe it could be a config boolean, and remove the need for a popup dialog entirely?

...on reviewing the code, I discovered [there's already a flag](https://github.com/nvim-telescope/telescope-frecency.nvim/blob/f67baca08423a6fd00167801a54db38e0b878063/lua/frecency/klass.lua#L136-L140) for suppressing the popups. Setting `db_safe_mode = false` seems to stand for "always say Yes", and with this config setting I can use the newest commit in the repo without the prefilled `A` problem!

tl;dr @fishman if you see this issue re-occur and want a workaround, try setting `db_safe_mode = false` in the frecency config.

**Comment by ChrisVilches** [comment]
@solson 

I used `db_safe_mode = false` and the problem (default value of `A` in the input) went away. The issue got fixed even after removing the flag `db_safe_mode = false`.

**Comment by solson** [comment]
> The issue got fixed even after removing the flag `db_safe_mode = false`.

I believe it would reoccur again the next time you have stale entries (i.e. files that were deleted) and frecency wants to show you the prompt to remove them. Setting it temporarily and then removing it again as you described basically just let you dismiss the prompt you already had.

**Comment by ippachi** [comment]
I also encountered this error.
I'm not sure if dressing.vim itself is the cause, but it seems to occur when used together with dressing.vim.
It might happen when using it alongside a plugin that configures vim.ui.select.

**Comment by pjezusek** [comment]
Adding `db_safe_mode = false` indeed solved the problem. I would add that it also helped me with something similar to #178. I had a problem where `code_actions` from nvim-lsp didn’t appear when I used Telescope with frecency. When I opened a file using the standard Telescope method or `:e`, it worked. So I assume this is something with `vim.ui.select`.
